lib/Echidna/Solidity.hs

```diff
@@ -99,7 +99,7 @@ loadSolidity filePath selectedContract = do
                   loadContract (vm ^. state . contract)
         loaded = execState load $ execState (replaceCodeOfSelf bc) vm
         abi = map (liftM2 (,) (view methodName) (map snd . view methodInputs)) . toList $ c ^. abiMap
-        (tests, funs) = partition (isPrefixOf "echidna_" . fst) abi
+        (tests, funs) = partition (isPrefixOf "deepstate_" . fst) abi
```

Comment:
I'm not sure how I feel about this prefix change. Seems like it would break _a lot_ of existing code. I'd probably prefer to keep it as-is unless there's more justification for such a breaking change

Reply:
I updated the prefix to standardize the property names following the configuration files used in zoo (for instance, [this one](https://github.com/trailofbits/smart-contracts-testing/blob/master/zoo/tokens/vulnerableToken/contracts/configManticore-WithOwner.yaml#L12))

Reply:
Let's consider moving this to the config then? I'd really like to avoid breaking peoples' existing tests, how would you feel about some kinda "`test_prefix`" field